sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.16.1 (Unreleased)
+## 12.17.0-beta.1 (Unreleased)
```

Comment:
Why the customized audience change is not included in the change log?

Reply:
It's already supported in previous version, it's not new for blob, we just added the same features for other packages..